web/src/compose.js

```diff
@@ -78,7 +83,7 @@ export function show_preview_area() {
     );
 }
 
-export function create_message_object() {
```

Comment:
This refactor can be in it's own prep commit

Reply:
I've separated it into its own commit